### ISSUE TITLE
Table of content is going to second for large text and gives spaces before start line

### ISSUE THREAD

**Issue by sumon01914** [mpdf/mpdf#1313]
I am struggling in lots of issue for table of content.  I am creating a large pdf with dynamically. It is working fine. Bu when text is large it is going to second line and gives some spaces before starting line although line are in same level(level1). I want to text alignment in one line. If it goes to second line for large text  then it will maintain the level. If text is for level1 then breaking text will start this level.  File attached for understanding. 
![Capture](https://user-images.githubusercontent.com/6293134/98902577-324b4500-24e0-11eb-84fe-9d328ce47d41.PNG)

**Comment by Klap-in** [comment]
Please share a small example of html and php code which reproduce your issue. Could you mention the software versions of your setup as well?

**Comment by sumon01914** [comment]
Hello, Thank for your reply. I am using mpdf version 8.0. Now it is solve by modify css. I always set text-indent from each level by 0em. But creating another issue. Page number is breaking the line. Attached file as proved with highlighted.
![Uploading 2.PNG…]()

**Comment by finwe** [comment]
Closing as per issue template warning. 

Feel free to open a new issue using the provided issue template with all available issue details, especially a code example to reproduce the issue more easily.
